esp/services/ws_workunits/ws_workunitsAuditLogs.cpp

```diff
@@ -1406,6 +1406,28 @@ void CWsWorkunitsSoapBindingEx::createAndDownloadWUZAPFile(IEspContext& context,
     request->getParameter("WhereSlow", zapInfoReq.whereSlow);
     request->getParameter("IncludeThorSlaveLog", zapInfoReq.includeThorSlaveLog);
     request->getParameter("ZAPFileName", zapInfoReq.zapFileName);
+
+    StringBuffer sendEmailStr, attachZAPReportToEmailStr, portStr;
+    request->getParameter("SendEmail", sendEmailStr);
+    zapInfoReq.sendEmail = atoi(sendEmailStr.str()) ? true : false;
+    if (zapInfoReq.sendEmail)
+    {
+        request->getParameter("AttachZAPReportToEmail", attachZAPReportToEmailStr);
+        zapInfoReq.attachZAPReportToEmail = atoi(attachZAPReportToEmailStr.str()) ? true : false;
+
+        request->getParameter("EmailSender", zapInfoReq.emailSender);
+        request->getParameter("EmailSubject", zapInfoReq.emailSubject);
+        request->getParameter("EmailBody", zapInfoReq.emailBody);
+        //Read maxAttachmentSize from setting.
+        zapInfoReq.maxAttachmentSize = wswService->zapEmailMaxAttachmentSize;
+        zapInfoReq.emailServer.set(wswService->zapEmailServer.get());
+        zapInfoReq.port = wswService->zapEmailServerPort;
+        zapInfoReq.emailTo.set(wswService->zapEmailTo.str());
```

Comment:
This looks like it's using a supplied destination

Reply:
The wswService->zapEmailTo is preconfigured. Please see line 402 in  esp/services/ws_workunits/ws_workunitsService.cpp.